src/FormBuilder/resources/js/form-builder/src/blocks/fields/payment-gateways/Edit.tsx

```diff
@@ -18,6 +20,12 @@ const GatewayItem = ({label, icon}: {label: string; icon: ReactNode}) => {
 };
 
 export default function Edit(props: BlockEditProps<any>) {
+    const isGatewayEnabled = (gateway: Gateway) => {
+        let enabled = applyFilters('givewp_form_builder_enabled_payment_gateway_' + gateway.id, [gateway.enabled]);
+        enabled = typeof enabled === 'undefined' ? true : enabled[0];
+        return enabled;
+    };
+
     const {gateways} = getFormBuilderWindowData();
```

Comment:
One more option we have here (that can replace or be used with the enabled filter) is something like this:

```
const gateways = applyFilters('givewp_form_builder_block_data_payment_gateways', data.gateways) as typeof data.gateways
```

That would offer more flexibility but maybe more risk of messing up the data. Depends on the complexity of the implementation.  With this you could do:

```
wp.hooks.addFilter('givewp_form_builder_block_data_payment_gateways', 'givewp', (gateways) => {
         return gateways.map((gateway) => {
             if (gateway.id === 'manual') {
                 gateway.enabled = false;
             }

             return gateway;
         });
    })
```

Reply:
Initially, I considered using this approach, but as you said it can get messed up if it's used in the wrong way. 

So, I preferred using a filter just to handle the `enabled` property as it would be a more straightforward way to do it - as we only need to handle this property right now. 

However, we can add this other filter later in the codebase if necessary.

Reply:
Sounds good 👍